manticore/ethereum.py

```diff
@@ -238,6 +242,30 @@ def will_evm_execute_instruction_callback(self, state, instruction, arguments):
             self.add_finding_here(state, 'Reachable SELFDESTRUCT')
 
 
+class DetectEtherLeak(Detector):
+    def will_evm_execute_instruction_callback(self, state, instruction, arguments):
```

Comment:
Should this be .. will _open_transaction..or should this include CREATE/DELEGATECALL?

Reply:
i had detecting `address.transfer()`s in mind when writing this. those always use calls. however i would guess that you're probably right that attackers could alternatively use other calls to steal ether. i think delegatecall could be plausible, less sure for create haha, that sounds pretty funky